.github/workflows/ci.yml

```diff
@@ -29,17 +29,6 @@ jobs:
         # instead of 14 GB) which is too little for us right now. Revisit when `dfr` commands are
         # removed and we're only building the `polars` plugin instead
         platform: [windows-latest, macos-13, ubuntu-20.04]
-        feature: [default, dataframe]
-        include:
-          - feature: default
-            flags: ""
```

Comment:
same comment about flags as below. i'll try to find the issue about winget and flags to make sure if it's this flags matters or not